components/about_build_modal/about_build_modal_cloud/about_build_modal_cloud.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Modal} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import MattermostLogo from 'components/widgets/icons/mattermost_logo';
+
+import './about_build_modal_cloud.scss';
+
+type Props = {
+    onHide: () => void;
+    config: any;
+    license: any;
+    show: boolean;
+    doHide: () => void;
+};
+
+export default function AboutBuildModalCloud(props: Props) {
+    const handleExit = () => {
+        props.onHide();
+    };
+
+    const config = props.config;
+    const license = props.license;
+
+    const title = (
+        <FormattedMessage
+            id='about.cloudEdition'
+            defaultMessage='Cloud'
+        />
+    );
+
+    const subTitle = (
+        <FormattedMessage
+            id='about.enterpriseEditionSst'
+            defaultMessage='High trust messaging for the enterprise'
+        />
+    );
+
+    //
+    const licensee = (
+        <div className='form-group'>
+            <FormattedMessage
+                id='about.licensed'
+                defaultMessage='Licensed to:'
+            />
+            {'\u00a0' + license.Company}
```

Comment:
For company we want to show the company information when they signed up for cloud and not the company of the license because in this case is always Mattermost. 

We can add this later when we have the endpoints to get Company information from CWS which we will have for Billing in the System Console Soft GA. Created a ticket https://mattermost.atlassian.net/browse/MM-28698